release/7-1/arm32v7/meta.json

```
@@ -7,8 +7,8 @@
     "Base64EncodePackageUrl": true,
     "UseAcr": true,
     "tagTemplates": [
-        "#psversion#-arm32v7-ubuntu-bionic-#fulltag#",
-        "#psversion#-arm32v7-ubuntu-18.04-#fulltag#",
+        "#psversion#-arm32v7-ubuntu-bionic",
+        "#psversion#-arm32v7-ubuntu-18.04",
         "arm32v7-ubuntu-#shorttag#"
     ],
     "shortTags": [
```

I think the shorttag list should also include "18.04" in addition to "bionic" so we can have images tagged as "arm32v7-ubuntu-18.04" and "arm32v7-ubuntu-bionic"
```suggestion
    "shortTags": [
     {"Tag": "18.04"},
```

---

fixed in all active channels